### ISSUE TITLE
handle directly calling API methods from global objects (`window`, `global`, `globalThis` )

### ISSUE THREAD

**Issue by 5punk** [amilajack/eslint-plugin-compat#92]
But the plugin doesn't reflect that.

`window.location.origin` [browser compatibility](https://developer.mozilla.org/en-US/docs/Web/API/Window/location).

**Comment by amilajack** [comment]
https://github.com/amilajack/eslint-plugin-compat/issues/72#issuecomment-288834016

**Comment by amilajack** [comment]
This is working now but only for the following cases:
```js
# Throws error:
location.origin();
# Does NOT throw error:
window.location.origin();
```

**Comment by kirly-af** [comment]
@amilajack Hi, are those errors willingly ignored, or is it a limitation that should be fixed? Making a PR for this does not seem to be too complicated.

**Comment by amilajack** [comment]
This was fixed in https://github.com/amilajack/eslint-plugin-compat/issues/92

You can get the changes by installing the latest version of the plugin:
```bash
yarn add --dev eslint-plugin-compat@next
npm i --save-dev eslint-plugin-compat@next
```
